src/notifications/notifications.service.ts

```diff
@@ -0,0 +1,104 @@
+import { Injectable } from '@nestjs/common';
+import { getMessaging } from 'firebase-admin/messaging';
+import { firebaseConfig } from 'src/configs/firebase/firebase.config';
+import { PrismaService } from 'src/prisma/prisma.service';
+import { UpdateNotificationDto } from './dto/update-notification.dto';
+import { NotificationDto } from './dto/notification.dto';
+import { User } from '@prisma/client';
+import * as firebase from 'firebase-admin';
+import * as path from 'path';
+
+firebase.initializeApp({
+  credential: firebase.credential.cert(
+    path.join(__dirname, '..', 'configs/firebase/firebase-admin.json'),
+  ),
+});
```

Comment:
It's better to create a file like `lib/firebase/firebase-admin.ts` or `src/modules/firebase/firebase-admin.ts` and initialize the app there